ci_files/Dockerfile-wf-standalone

```diff
@@ -0,0 +1,8 @@
+FROM hawkular/wildfly-hawkular-agent:latest
```

Comment:
We need to move away from wildfly-hawkular-agent and use the javaagent instead.

Reply:
> There is a Travis error that pops up too frequently: https://travis-ci.org/ManageIQ/manageiq-providers-hawkular/jobs/275600024#L2615
> 
> Probably we should skip the live tests in Travis and only use them in our local envs?
> I think that Server broke connection is related to websocket-client-simple used in the hawkular-client
> @rubenvp8510 might know more about this, as he was working on replacing it hawkular/hawkular-client-ruby#205

This not seems to be related to the WebSocket library, at least not directly, that library has a problem because it closes the socket before kills the polling thread and sometimes try to read from a closed socket (with getc) and that causes a segfault) this is not the case.

Here for some reason, the server is ending the connection or there are some communication issues before ending the request.

Reply:
@josejulio is this good to merge?

Reply:
wildfly-hawkular-agent won't be supported for inventory v4, we should use `hawkular/wildfly-hawkular-javaagent` The config is almost the same (yaml instead of xml).

Hawkular-services already uses the javaagent. (even in inventory v3)